tests/test-env-py39.yml

```diff
@@ -49,7 +49,6 @@ dependencies:
   - moto
   - deepdiff
 
-
   # for docs
   - sphinx
```

Comment:
Don't do it now, but, omg, testing everywhere is such a mess. I'm sure we don't need sphinx here...